www_src/lib/keyboard.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  simulateTabPress: function () {
+    var current = document.activeElement;
+    var elements = document.getElementsByTagName('input');
+    var idx = parseInt(current.getAttribute('tabIndex'), 10) + 1;
+
+    for (var i = 0; i < elements.length; i++) {
+      var t = parseInt(elements[i].getAttribute('tabIndex'), 10);
+      if (t === idx) {
```

Comment:
technical nit: this isn't true. In HTML, DOM ordering matters too, so you can have 100 elements with the same tab index, and the "next" element to pick in absence of a higher tabindex is simply the next in line. That's way more than we probably need, but it's worth noting in the code that this is purely for testing, and an incomplete implementation